src/screens/Subscriptions/Subscriptions.style.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import { Typography, Box } from '@material-ui/core';
+
+export const Wrapper = styled(Box)`
+  max-width: 700px;
+  width: 100%;
+  margin: 80px auto 100px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  padding: 1.2rem;
+
+  @media (min-width: 600px) {
+    align-items: center;
+    margin: 100px auto 125px;
+    min-height: 50vh;
+  }
+`;
+
+export const Header = styled(Typography)`
+  ${props => props.theme.fonts.regularBookBold};
+  font-size: 2rem;
+  margin-bottom: 2.5rem;
+  line-height: 1.2;
+`;
+
+export const BodyCopy = styled(Typography)`
+  ${props => props.theme.fonts.regularBook};
+  font-size: 1.1rem;
+  margin-bottom: 1.75rem;
+  line-height: 1.4;
```

Comment:
as i start learning the codebase/design system here - have we considered having standard paddings/sizes and such for typography? what's the general guidelines, if any, for custom values for components?

(just thinking/wondering out loud btw, nothing actionable for this diff)

Reply:
(i'm thinking ahead to BBE and other work - things like, say, `600px` in the `@media (min-width: 600px)` above could be constants for consistency across the codebase.)

Reply:
Heh, we definitely should (in the BBE codebase especially)... but the covid codebase has evolved a bit haphazardly and it's taken us a while to get around to unifying styles.  So I don't _think_ we have any typography styles ready to be reused here.  We do for e.g. buttons, but not so much for typography.  Though @pnavarrc or @chasulin may know better.  I don't think it's worth too much effort to unify at this point, but we should for the BBE codebase.

Reply:
600px breakpoint [is indeed a constant](https://github.com/covid-projections/covid-projections/blob/develop/src/assets/theme/sizes.tsx#L4)

as for unifying spacing, im starting to clean that up a little bit [in the PR](https://github.com/covid-projections/covid-projections/pull/3577). as michael mentioned, [buttons are newly unified](https://github.com/covid-projections/covid-projections/blob/develop/src/components/ButtonSystem/MainButtons.style.tsx)

Reply:
but yeah, allllll for unifying spacing/sizing/breakpoints/etc in the BBE code from the get go 💫💫

Reply:
Ah, nice. Switched over to materialSMBreakpoint constant!

Reply:
also [started to do this for fonts](https://github.com/covid-projections/covid-projections/blob/develop/src/assets/theme/fonts.ts), but just font family + font weight

Reply:
ooo excellent @chasulin!